.github/workflows/sanity-test.yml

```diff
@@ -1,11 +1,19 @@
 name: Sanity Test
 
-on: pull_request
+on: pull_request_target
 
 jobs:
   sanity-test:
     runs-on: ubuntu-latest
     steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
+        with:
+          # this might remove tools that are actually needed,
+          # if set to "true" but frees about 6 GB
+          tool-cache: false
+          
+          docker-images: false
```

Comment:
this is required, otherwise the job fails on pulling the build pipeline images due to `No space left on device` error